tools/firmware.toit

```diff
@@ -236,44 +255,61 @@ properties_update parsed/cli.Parsed [block] -> none:
     if properties: envelope.entries[AR_ENTRY_PROPERTIES] = json.encode properties
 
 extract_cmd -> cli.Command:
+  flags := AR_ENTRY_FILE_MAP.map: | key/string value/string |
```

Comment:
Alternative is to make each of those options:
`extract --firmware.bin=xyz --partitions.csv=...`

That would also avoid the exclusivity check in line 277.

Reply:
Let's discuss the options next week. I'm all for further tweaks.